extensions/amp-ad/0.1/test/test-amp-ad-api-handler.js

```diff
@@ -65,17 +68,80 @@ describe('amp-ad-api-handler', () => {
       }));
     });
 
-    it('should resolve startUp() when render-start API is called', () => {
-      expect(iframe.style.visibility).to.equal('hidden');
-      iframe.postMessageToParent({
-        sentinel: 'amp3ptest' + testIndex,
-        type: 'render-start',
+    describe('should resolve startUp', () => {
```

Comment:
`describe('method startUp(), ...`

We normally have noun phrases in `describe` and a should-clause in `it`

Here is [a good read](http://samwize.com/2014/02/08/a-guide-to-mochas-describe-it-and-setup-hooks/) for how to use describe-it clause in Mocha test.